btree/btree.c

```diff
@@ -2690,8 +2678,10 @@ static int64_t btree_put_alloc_phase(struct m0_btree_op *bop)
 		} else {
 			node_op_fini(&oi->i_nop);
 			oi->i_used = bop->bo_arbor->t_height - 1;
+			if (bop->bo_flags & BOF_LOCKALL)
+				lock_op_unlock(tree);
```

Comment:
same comment as above.

Reply:
Need to see if we really want to add lock_acquired parameter for level_cleanup

Reply:
please refer above comment